app/src/main/java/com/epfl/drawyourpath/database/FireDatabase.kt

```diff
@@ -335,6 +374,23 @@ class FireDatabase : Database() {
             return BitmapFactory.decodeByteArray(tabByte, 0, tabByte.size)
         }
     }
+
+    /**
+     * Helper function to obtain the friends list from the database of the user
+     * @param data the data snapshot containing the user List
+     * @return a list containing the userIds of the friends
+     */
+    private fun transformFriendsList(data: DataSnapshot?): List<String>{
+        if(data == null){
+            throw java.lang.Error("The friends list is not present on the database.")
```

Comment:
Use Kotlin types.
Also, if the user has no friend, we don't need to throw an error.